apps/fishing-map/features/timebar/TimebarActivityGraph.utils.ts

```diff
@@ -0,0 +1,74 @@
+import { Feature } from '@globalfishingwatch/maplibre-gl'
+import { getTimeSeries } from '@globalfishingwatch/fourwings-aggregate'
+import { quantizeOffsetToDate, Interval } from '@globalfishingwatch/layer-composer'
+import { DataviewFeature } from 'features/map/map-sources.hooks'
+
+type TimeseriesParams = {
+  features: Feature[]
+  quantizeOffset: number
+  numSublayers: number
+  interval: Interval
+  visibleSublayers: boolean[]
+}
+export const getTimeseries = ({
+  features,
+  quantizeOffset,
+  numSublayers,
+  interval,
+  visibleSublayers,
+}: TimeseriesParams) => {
+  if (!features || !features.length) {
+    return []
+  }
+
+  // let prevMaxFrame: number
+  const { values, maxFrame } = getTimeSeries(features as any, numSublayers, quantizeOffset)
+
+  // TODOTODOTODOTODOTODOTODOTODOTODOTODOTODOTODOTODO
+  // TODO!! ask Erik what does it do  TODOTODOTODOTO!!
+  // TODOTODOTODOTODOTODOTODOTODOTODOTODOTODOTODOTODO
+
+  // const valuesTimeChunkOverlapFramesFiltered = prevMaxFrame
+  //   ? values.filter((frameValues) => frameValues.frame > prevMaxFrame)
```

Comment:
I can explain. Time chunks overlap, ie if you load "1 year" it actually loads 1 year + 3 months (needed for aggregation). So you have 2016 and 2017 as active chunks, you end up with the same values in both chunks for the period january - march 2017. This ensures that when looking at the 2017 chunk, you only use data starting from april 2017.